requirements

```diff
@@ -1,4 +1,5 @@
-bitstring
+bitstring==3.1.9
```

Comment:
At some point bitstring changed something and broke my stuff. I don't think it's a big deal, but it does need some attention.